src/api/projectApi.ts

```diff
@@ -0,0 +1,29 @@
+import Axios from 'axios';
+import { Dispatch } from 'redux';
+
+import { addNotification } from '../redux/actions/notificationActions';
+import i18n from '../translations/i18n';
+import { StatusCode } from '../utils/constants';
+import { getIdToken } from './userApi';
+
+export enum ProjectsApiPaths {
+  PROJECTS = '/cristin-projects',
+}
+
+export const searchProjects = async (query: string, dispatch: Dispatch) => {
+  try {
+    const idToken = await getIdToken();
+    const response = await Axios.get(`${ProjectsApiPaths.PROJECTS}?${query}`, {
```

Comment:
bedre å skrive `title=` her isteden for i ProjectSearch.tsx?
`${ProjectsApiPaths.PROJECTS}?title=${query}`

Reply:
Alternativt lage et query-object, men siden vi kun bruker title pr nå  så renamer jeg heller funksjonen til searchByTitle